bibliopixel/remote/server.py

```diff
@@ -0,0 +1,93 @@
+from flask import Flask, jsonify, send_from_directory
+from werkzeug.serving import run_simple
+import threading
+import queue
+import os
+import logging
+from .. import log
+
+# I want to keep this to supress the "Running on" message
```

Comment:
I made a comment about a feature flag here, but it seemed to get lost:

Something I became aware of in my previous project is that if you comment code out, you lose all your automatic checks on it.

Consider instead using a "feature flag" sort of idea:

    USE_WERK_LOG = False
    if USE_WERK_LOG:
        werk_log = logging.getLogger('werkzeug')
        werk_log.setLevel(logging.ERROR)
        # comments go here.